www/config.yml

```diff
@@ -129,6 +129,14 @@ chapters:
       email: chicago@pyladies.com
       twitter: PyLadiesChicago
       meetup: Chicago-PyLadies
+  
+    - name: Dallas, TX
+      meetup_id: 2292131
```

Comment:
Sheila - it's difficult to find. You have to go to the dev site of Meetup and use the console to do an API request using the "groups" endpoint. Let me know if you want further instruction.

Side note: a script will be added to the pyladies package to easily make that API request:)

Fat fingered from my iPhone

> On Apr 28, 2015, at 7:58 PM, Sheila Allen <notifications@github.com> wrote:
> 
> In www/config.yml:
> 
> > @@ -129,6 +129,14 @@ chapters:
> >        email: chicago@pyladies.com
> >        twitter: PyLadiesChicago
> >        meetup: Chicago-PyLadies
> > +  
> > +    - name: Dallas, TX
> > +      meetup_id: 2292131
> Where does this ID come from?
> 
> —
> Reply to this email directly or view it on GitHub.
>